test/to-markdown/maths/block/output.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Since math block don't really exist (inline math inside a paragraph with a property `inline: false`), we wrap math blocks with one newline (in case where multiple math nodes are added in the same paragraph).

The previous test runner was trimming these newlines.